include/Core/Config/Exceptions.hpp

```diff
@@ -0,0 +1,17 @@
+#include <Exception.hpp>
+
+#include <string_view>
+
+namespace obe::Config::Exceptions
+{
+    class InvalidVersionNumber : public Exception
```

Comment:
I'd rename it `InvalidVersionFormat` based on the way you use it ?

Reply:
True, I will adapt this.